src/tools/pkcs11-tool.c

```diff
@@ -6255,8 +6255,6 @@ static int test_decrypt(CK_SESSION_HANDLE sess)
 			continue;
 		}
 
-		printf("\n");
-
```

Comment:
I do not think this is the correct fix. The above messages refer to the objects that are skipped for some reason (all followed with continue). If we get here, we start testing the key with different mechanisms and it needs to follow on a new line.